### DISCUSSION TITLE
How can I remove the "Recent posts" section?

### DISCUSSION THREAD

**Discussion by tomchor** [mmistakes/minimal-mistakes#3624] [Q&A] [answered]
My home page ends with a "Revent posts" section that I'm not including in `index.md`. 

I'm only using pages and don't have any posts, so I don't know why that's there. I've looked through the docs but I couldn't figure it out!

How can I remove that section/header?

Thanks!

**Comment by iBug** [comment, marked as answer]
Use `layout: single`. The `home` layout is designed to be a "menu".

**Reply by tomchor** [reply]
That's right! I had changed the default layout to `single` so I thought it applied to the home page, but it was indeed being set to `home`. Thanks for the tip!
